admin/includes/languages/english/alert_page.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
  * @package admin
- * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
- * @version $Id: Author: DrByte  Sun Aug 2 22:46:14 2015 -0400 Modified in v1.5.5 $
+ * @version $Id: Author: DrByte  Sun Aug 2 22:46:14 2015 -0400 Modified in v1.5.5 $
  */
 
 define('HEADING_TITLE', 'Warning!');
-define('ALERT_PART1', 'Warning: You cannot access the admin until you have');
+define('ALERT_PART1', 'Warning: You cannot access the admin area until you have');
 define('ALERT_RENAME_ADMIN', 'renamed the admin folder.');
-define('ALERT_REMOVE_ZCINSTALL', 'deleted the zc_install folder.<br />(Use your FTP program or your hosting control panel.)');
+define('ALERT_REMOVE_ZCINSTALL', 'deleted the zc_install folder.<br>(Use your FTP program or your hosting control panel.)');
 define('ADMIN_RENAME_FAQ_NOTE', 'Help for renaming the admin folder can be found here');
-define('ALERT_PART2', 'Then, to access your admin area, type the admin URL into your browser, ie: <u>http://www.your_site.com/YourAdminFolder/</u> ');
+define('ALERT_PART2', 'Then, to access your admin area, type the admin URL into your browser, ie: <strong>http://www.your_site.com/YourAdminFolder/</strong> ');

```

Comment:
Why the removal of underline?

Reply:
It was purely visual, looked odd, the underscores on top of/under the underline.